qiskit_nature/second_q/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_uvcc_factory.py

```diff
@@ -91,46 +95,52 @@ def __init__(
         self._initial_point = initial_point if initial_point is not None else VSCFInitialPoint()
         self._ansatz = ansatz
 
+        if "estimator" not in kwargs:
+            kwargs["estimator"] = Estimator()
+        if "optimizer" not in kwargs:
+            kwargs["optimizer"] = SLSQP()
```

Comment:
Do we want to imply these defaults or should we rather mirror the `VQE` signature and require both of these as positional arguments?